### PR TITLE
Allow resetting user's password

### PR DESCRIPTION
**Pull request by siemiatj**
Related to #1878 

TODO:
- [x] avatar is loaded in weird format
- [x] handle invalidated tokens. Now it just fails silently caught by the global app error.

### CODE REVIEW

**Comment by metas-ts** [comment]
hi @siemiatj,
pls check out the jenkins & codacy issues

**Comment by siemiatj** [comment]
This needs changes anyways, already talked with Teo about that.

File: src/containers/Login.js [hunk 1]
```diff
@@ -65,12 +66,14 @@ class Login extends Component {
   };
 
   render() {
-    const { redirect, auth } = this.props;
+    const { redirect, auth, splat, token } = this.props;
```
[the remark this answers is not in the document]
Comment (siemiatj) [reply src/containers/Login.js]: are you F'n stupid or what ? IT IS THERE ! What a useless application this is.

[the remark this answers is not in the document]
Comment (siemiatj) [reply src/containers/Login.js]: what a useless piece of c*(p. Anyways all props are there :)

**Pull request merged**